web-ng/package.json

```diff
@@ -7,13 +7,15 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
+    "eslint": "eslint .",
+    "prettier": "prettier --single-quote --trailing-comma es6 --print-width 80 --write --no-bracket-spacing **/*.js",
     "e2e": "ng e2e",
     "check": "gts check",
     "clean": "gts clean",
     "compile": "tsc -p .",
     "fix": "gts fix",
     "prepare": "npm run compile",
-    "pretest": "npm run compile",
+    "pretest": "eslint --fix .",
```

Comment:
This would actually modify the code when running tests - is this intentional?

Reply:
Let's revert this. I assumed that tests will be run locally everytime